### PR TITLE
[WIP] P11 develop 2

### PR DESCRIPTION
**Pull request by agruzinov**
Small changes to the P11 HO after testing.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mxcubecore"
-version = "1.76.0"
+version = "1.77.0"
```
Comment (fabcor-maxiv) [inline pyproject.toml]: As far as I understood the process, that should not be done from within a pull request.

```suggestion
version = "1.76.0"
```

**Review by fabcor-maxiv — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (marcus-oscarsson) [reply pyproject.toml]: Its indeed strange that the PR includes a previous commit, from the CI (#4f6c5d9)

**Comment by agruzinov** [comment]
Is there a cleaner way to incorporate all the changes and having a fork without having all the commits that happened in between after rebase? 

For tnow it is like this:

There is a branch of core/qt under mu user.
Than:

-Synchronise all the changes that happened in between with the develop branch through the web interface. 
Than on the develop maschine:
git checkout develop
git pull (with rebase)

than:

git checkout p11_develop_2
git rebase develop
git push

Rhetorical question: what I'm doing wrong? :)

Or it is easier just to to work on the main repo than?

Any comments will be appreciated? Sorry for the basic question, but I think I've lost the understanding of the git behaviour.

Cheers!



git branch p11_develop_2

File: pyproject.toml (hunk 1 above)
Comment (agruzinov) [reply pyproject.toml]: That happens if I get the latest updated from the upstream develop to agruzinov/develop through github web interface.

**Comment by agruzinov** [comment]
Resolved through another PR#905. Closing.

**Pull request closed without merging**